__tests__/app.test.js

```diff
@@ -270,4 +270,75 @@ describe('API', () => {
             })
         });
     });
+
+    describe('GET /API/ARTICLES QUERIES', () => {
+        test('Status 200 - returns back an array of articles which have topic mitch', () => {
+            return request(app)
+                .get("/api/articles?topic=mitch")
+                .expect(200)
+                .then(({ body }) => {
+                    body.articles.forEach((article) => {
+                        expect(article.topic).toBe("mitch");
+                      });
+                })
+        });
+        test('Status 200 - returns back an array of articles which have topic cats', () => {
+            return request(app)
+                .get("/api/articles?topic=cats")
+                .expect(200)
+                .then(({ body }) => {
+                    body.articles.forEach((article) => {
+                        expect(article.topic).toBe("cats");
+                      });
+                })
+        });
+        test('Status 200 - returns back an array of sorted articles by votes', () => {
+            return request(app)
+                .get("/api/articles?sort_by=votes")
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.articles).toBeSortedBy("votes", { descending: true });
+                })
+        });
+        test('Status 200 - returns back an array of sorted articles by created_at', () => {
+            return request(app)
+                .get("/api/articles?order=asc")
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.articles).toBeSortedBy("created_at", { ascending: true });
+                })
+        });
+        test('Status 200 - returns back an empty array of articles when the value of topic does not exist in DB', () => {
```

Comment:
I think here if the topic doesnt exist in the database we could return an error to the user to let them know that topic doesn't exist